android/sdk/src/main/java/com/taobao/weex/dom/action/CreateFinishAction.java

```diff
@@ -18,15 +18,36 @@
  */
 package com.taobao.weex.dom.action;
 
+import com.taobao.weex.LayoutFinishListener;
 import com.taobao.weex.WXSDKInstance;
+import com.taobao.weex.WXSDKManager;
 import com.taobao.weex.common.WXRenderStrategy;
+import com.taobao.weex.common.WXThread;
+import com.taobao.weex.dom.DOMActionContext;
 import com.taobao.weex.dom.RenderActionContext;
 
 /**
  * Created by sospartan on 02/03/2017.
  */
 
 final class CreateFinishAction extends AbstractLayoutFinishAction {
+
+  @Override
+  public void executeDom(DOMActionContext context) {
+    super.executeDom(context);
+    //add by chuyi
```

Comment:
i want 'save cached instance' when layout finished. so i add `LayoutFinishListener`, when createFinish action invoked, the listener will fired too.